src/App.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from 'react';
+import Page from './Page';
+
+export default function App() {
+  const [state, setState] = useState({
+    count: 0,
+  });
+
+  const { count } = state;
+
+  const handleClick = ({ increment }) => {
```

Comment:
```suggestion
  const handleClick = (increment) => {
```
이렇게 한다면 Page 컴포넌트에서도 조금 더 간결하게 사용 가능할 것 같아요~